src/Core/Addon/Module/ModuleInterface.php

```diff
@@ -32,6 +32,8 @@ interface ModuleInterface extends AddonInterface
 {
     public function onInstall();
```

Comment:
Maybe you can strong type all the other functions, since there is already a BC in this interface